### PR TITLE
Providing a more direct and helpful pip command

### PR DESCRIPTION
**Pull request by dinagraves**
Showing how to install the package from this repo using pip3.

### CODE REVIEW

**Review by wkozlik — APPROVED** [review]
Thank you for this improvement.

File: README.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ Python 2.7 and 3.4+
...
 
-If the python package is hosted on Github, you can install directly from Github
+To install the package from this repository:
 
 ```sh
-pip install git+https://github.com/GIT_USER_ID/GIT_REPO_ID.git
+pip3 install git+https://github.com/grafeas/client-python.git#egg=Grafeas --user
 ```
-(you may need to run `pip` with root permission: `sudo pip install git+https://github.com/GIT_USER_ID/GIT_REPO_ID.git`)
+Note: If running on Linux, you need to escape the `&` as shown above.  Other systems may not require the escape character.
```
Comment (nyc) [inline README.md:22]: I think this is no longer relevant since the instructions have been updated.

**Review by nyc — COMMENTED** [review]
LGTM besides minor extraneous line in the README.

File: README.md (hunk 1 above)
Comment (dinagraves) [reply README.md:22]: Good point! Thanks.

**Review by nyc — APPROVED** [review]

**Pull request merged**